Xamarin.Forms.Platform.WPF/Xamarin.Forms.Platform.WPF.csproj

```diff
@@ -1,57 +1,10 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{140BC260-8B15-4D3A-B1B0-DDD8072918CC}</ProjectGuid>
-    <OutputType>library</OutputType>
-    <RootNamespace>Xamarin.Forms.Platform.WPF</RootNamespace>
-    <AssemblyName>Xamarin.Forms.Platform.WPF</AssemblyName>
-    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <ProjectTypeGuids>{60dc8134-eba5-43b8-bcc9-bb4bc16c2548};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
-    <WarningLevel>4</WarningLevel>
-    <TargetFrameworkProfile />
+    <TargetFrameworks>net461;netcoreapp3.0</TargetFrameworks>
+    <UseWPF>true</UseWPF>
+    <UseWindowsForms>true</UseWindowsForms>
```

Comment:
This line is for Windows Forms applications, in this case it would not be necessary.

Reply:
` <Reference Include="System.Windows.Forms" />`
is referenced in the old csproj. I think it can no longer get referenced in the new project (did not work for me alteast) and adding the UseWindowsForms resolves the dependency errors.